fastplotlib/graphics/line_collection.py

```diff
@@ -30,6 +29,41 @@ def __init__(
             *args,
             **kwargs
     ):
+        """
+        Create a Line Collection
+
+        Parameters
+        ----------
+        data: list of array-like
+            List of line data to plot, if elements 2D must be of shape [n_points, 2], if elements
```

Comment:
If it's a `list` each element must be a 1D, 2D or 3D numpy array. If it's a 2D numpy array it's interpreated as `[y_vals, n_lines]`, like when we pass in a 2D array of temporal components it does a line for each component. It's diff from `LineGraphic`